ui/panels/dream_texture.py

```diff
@@ -365,6 +365,4 @@ def draw(self, context):
                 for i, line in enumerate(e.args[0].split('\n')):
                     error_box.label(text=line, icon="ERROR" if i == 0 else "NONE")
                 e._draw(prompt, context, error_box)
-            except Exception as e:
-                print(e)
```

Comment:
These exceptions can be safely propagated since its at the end of the draw function, now it can properly show traceback info on errors it encounters.